src/lib/hooks/usePose.svelte.ts

```diff
@@ -17,7 +16,7 @@ export const usePose = (name: () => string | undefined, parent: () => string | u
 	const logs = useLogs()
 	const { refreshRates } = useMachineSettings()
 	const partID = usePartID()
-	const motionClient = useMotionClient()
```

Comment:
We'll want to remove the "motion client" input and related infrastructure as well since it's not applicable any more